public/javascripts/components/SearchBar.js

```diff
@@ -1,32 +1,82 @@
+"use strict";
+/**
+ * Search bar will have the input form so the user can filter the lectures he wants to watch
+ * TODO: complete the style of the component
+ */
+
 import React from "react";
 import Chance from "chance";
 const chance = new Chance();
+import CourseActions from "../actions/CourseAction";
```

Comment:
keep local imports at the top and remote at the bottom and separate them by a new line